packages/vscode-extension/src/extension.ts

```diff
@@ -63,8 +63,8 @@ export function deactivate(context: ExtensionContext): undefined {
 export async function activate(context: ExtensionContext) {
   handleUncaughtErrors();
 
-  if (Platform.OS !== "macos" && Platform.OS !== "windows") {
-    window.showErrorMessage("Radon IDE works only on macOS and Windows.", "Dismiss");
+  if (Platform.OS !== "macos" && Platform.OS !== "windows" && Platform.OS !== "linux") {
+    window.showErrorMessage("Radon IDE works only on macOS and Windows and Linux.", "Dismiss");
```

Comment:
```suggestion
    window.showErrorMessage("Radon IDE works only on macOS, Windows and Linux.", "Dismiss");
```
Although, I wonder if we still need this error.